### DISCUSSION TITLE
Styling taking a while to load

### DISCUSSION THREAD

**Discussion by fenixpsicologia** [filamentphp/filament#5702] [Help] [answered]
Hello, I'm new using the filament and one of the things I'm having trouble solving is the style that apparently takes a while to load, in the video below you can see that some form fields are not stylized and then return to normal.

I'm currently in a development environment. I may have missed some point in the installation or configuration, I would appreciate any tips on how to solve this.

https://streamable.com/t0aqwn

**Comment by danharrin** [comment, marked as answer]
I'm sorry but it's a matter of 1-5ms, I don't think that's "a while" at all? It's JavaScript initialisation, not anything to do with CSS
